recipes/adc-streaming/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "adc-streaming" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "c1ab8aee52684ff1f1c5ae469a760a435d6c45836ead4e9a97423a4893cb92d6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/adc-streaming-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - setuptools-scm
+    - importlib-metadata  # [py < 38]
+    - python-confluent-kafka
+    - tqdm
+
+  run:
+    - python
+    - certifi >=2020.04.05.1
+    - dataclasses  # [py < 37]
+    - importlib-metadata  # [py < 38]
```

Comment:
`noarch: python` does not work with selectors.
It is safe to remove the selectors. `dataclasses` will install an empty package for python >=3.7 (see https://conda-forge.org/docs/maintainer/knowledge_base.html#empty-python-packages).
My understanding is that `importlib-metadata` can be installed with any version of Python.